lib/pure/random.nim

```diff
@@ -121,15 +121,19 @@ else:
 proc rotl(x, k: Ui): Ui =
   result = (x shl k) or (x shr (Ui(64) - k))
 
+template randState*(): untyped =
+  ## Makes the default Rand state accessible from other modules.
+  state
+
 proc next*(r: var Rand): uint64 =
   ## Computes a random ``uint64`` number using the given state.
   ##
   ## See also:
-  ## * `rand proc<#rand,Rand,Natural>`_ that returns an integer between zero and
+  ## * `rand proc<#rand,Natural,Rand>`_ that returns an integer between zero and
```

Comment:
Someone who knows, is this link correct?

Reply:
FYI: don't do co skip for doc changes in the future :)

Reply:
it doesn't work anyway.